### DISCUSSION TITLE
Maximum extent for importing OSM data

### DISCUSSION THREAD

**Discussion by tgestabrook** [domlysz/BlenderGIS#504] [Q&A]
I am trying to import buildings from OSM for a city-scale project, but I am getting a "too large extent" error. I couldn't find any info in the documentation about what the maximum extent for importing OSM data is, or if there is a way to temporarily shrink the working region to import fewer buildings at one time. 

Any help is appreciated, and thanks for making such a great extension.

**Comment by yaofuzhou** [comment]
I happen to have looked this up days ago and the answer can be found at
[https://github.com/domlysz/BlenderGIS/issues/160](https://github.com/domlysz/BlenderGIS/issues/160)
or this specifically
[https://github.com/domlysz/BlenderGIS/issues/160#issuecomment-849438841](https://github.com/domlysz/BlenderGIS/issues/160#issuecomment-849438841)
